gh-pages-src/pages/Docs.vue

```diff
@@ -188,6 +188,11 @@ export default {
         name: 'after-items-fixed-absolute',
         scopeProperties: {},
         description: 'Define a custom error appearance (fixed, absolute)'
+      },
+      {
+        name: 'select-text-on-focus',
```

Comment:
These are slots, not props. Props go to the documentation automatically. 😄

Reply:
Cleaned up Docs.vue 😃